rust/kcl-lib/src/unparser.rs

```diff
@@ -883,7 +883,7 @@ lazy_static::lazy_static! {
         import_file_extensions
     };
 
-    pub static ref RELEVANT_EXTENSIONS: Vec<String> = {
+    pub static ref RELEVANT_FILE_EXTENSIONS: Vec<String> = {
```

Comment:
If we're just having one global list of extensions, it probably doesn't make sense for it to live in the formatter. Maybe just keep it in lib.rs if there's not a better place

Reply:
will do!